tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/otextension/BristolOt.java

```diff
@@ -0,0 +1,61 @@
+package dk.alexandra.fresco.tools.ot.otextension;
+
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+import dk.alexandra.fresco.tools.ot.base.Ot;
+
+/**
+ * Container class for a protocol instance of Bristol OTs.
```

Comment:
I am not sure what a Bristol OT is. Can you be more specific? Possibly a link to a paper.